service/siteService.js

```diff
@@ -78,3 +89,35 @@ transformHours = function(time) {
 	}
 }
 
+transformPublicTransit = function(publicTransitOptions) {
+	return publicTransitOptions ? getPublicTransit(publicTransitOptions) : undefined
+}
+
+getPublicTransit = function(publicTransitOptions) {
+	let options = []
+	if(publicTransitOptions) {
```

Comment:
also, no real error handling right now :/

Reply:
But added some filtering as  workaround....is that weird?

Reply:
I missed this when I was scrolling before (I found it in the email notification thread), but I don't think that's weird at all.  It's just validation logic before you do something, like checking for `null`.

That said, if there is an issue it would be worth logging (although since we don't have any monitoring infrastructure, I don't know where it would go).

Reply:
Actually, I take that back.  If this is on Heroku, you could get to it by checking the `heroku logs` and then filtering out non-relevant lines.

It wouldn't alert proactively, but it would be informative if something were to happen.